Tests/ConfidenceTests/ConfidenceValueTests.swift

```diff
@@ -0,0 +1,143 @@
+import Confidence
+import XCTest
+
+final class ConfidenceConfidenceValueTests: XCTestCase {
```

Comment:
maybe `ConfidenceValueTests`?

Reply:
This came from main and wasn't part of Storage PR. We should rename it separately 👍